src/components/templates/UiQuestion/UiQuestion.vue

```diff
@@ -295,6 +307,7 @@ const defaultProps = computed(() => {
       icon,
       ...props.iconInfoAttrs,
     },
+    iconWhyAttrs: { ...props.iconWhyAttrs },
```

Comment:
icon:info it shouldn't be added by default?

Reply:
I think an icon shouldn't be default because it should be visible only for sex question. The rest of the questions have a rationale button without an icon:
![image](https://github.com/user-attachments/assets/6e8d0410-e0c6-42d2-a36a-1eed4e01c39c)